### PR TITLE
Feature/{chimereucheya}/Article page

### PR DESCRIPTION
**Pull request by Merdoth**
#### What does this PR do?
-Application Article page

#### Description of Task to be completed?
- Shows the application article page which gives full insight into the type of articles published by lifeAid.

#### How should this be manually tested?
- run the application using `npm run dev` visit `http://localhost:3000/article`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: Feature
- story title: Article page

#### Screenshots (if appropriate)
N/A

#### Question
N/A

### CODE REVIEW

File: components/Article/ArticlePage.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
```
Comment (enyeInc) [inline components/Article/ArticlePage.js]: content here can be mapped

File: components/Article/ArticlePage.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
```
Comment (enyeInc) [inline components/Article/ArticlePage.js:111]: all of this content can created through an array mapping

File: components/Article/ArticlePage.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                <p className="pagination"> 1  |  2  |  3  |  4  |  ...</p>
```
Comment (enyeInc) [inline components/Article/ArticlePage.js]: use antd list component to handle this list of contents and the pagination logic

File: components/Service/ServicePage.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Emergency Ambulance
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (enyeInc) [inline components/Service/ServicePage.js]: map over all of these contents

File: components/Service/ServicePage.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+                                Opportunity To Ask Doctors
+                                <br />
+                                {' '}
+                                Questions
+                            </p>
+                            <p className="service-text2">{HELP_DESK}</p>
+                        </span>
+                    </div>
+                </div>
+            </div>
```
Comment (enyeInc) [inline components/Service/ServicePage.js]: map over these contents

**Comment by enyeInc** [comment]
@Merdoth Your code overall looks good, you just have to adopt the concept of DRY (Don't Repeat Yourself). If you find yourself writing what looks like the same code more than one, you need to abstract that out into a function or a data structure of some kind

File: components/Article/ArticlePage.js (hunk 3 above)
Comment (Merdoth) [reply components/Article/ArticlePage.js]: Done

File: components/Service/ServicePage.js (hunk 4 above)
Comment (Merdoth) [reply components/Service/ServicePage.js]: Will do that once I start connecting it to the api directly

File: components/Article/ArticlePage.js (hunk 3 above)
Comment (Merdoth) [reply components/Article/ArticlePage.js]: Will do that once I start connecting it to the api directly

File: components/Article/ArticlePage.js (hunk 2 above)
Comment (Merdoth) [reply components/Article/ArticlePage.js:111]: Will do that once I start connecting it to the api directly

File: components/Service/ServicePage.js (hunk 5 above)
Comment (Merdoth) [reply components/Service/ServicePage.js]: Will do that once I start connecting it to the api directly

File: components/Article/ArticlePage.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import Head from 'next/head';
+import 'antd/dist/antd.css';
+import { Pagination } from 'antd';
+import NavigationBar from '../Common/NavigationBar';
+import { REPORT } from './constants';
+import { HOME_STRINGS } from '../constants';
```
Comment (EngrEric) [inline components/Article/ArticlePage.js:7]: keep all the constants for this component in the same place `import { HOME_STRINGS, REPORT, SERVICE_STRING  } from './constants';`

File: components/Article/ArticlePage.js [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+import { SERVICE_STRING } from '../Service/constants';
+
+import './article.scss';
+
+const {
+    REPORT_TEXT,
+} = REPORT;
```
Comment (EngrEric) [inline components/Article/ArticlePage.js:14]: `const { REPORT_TEXT } = REPORT;`

File: components/Article/ArticlePage.js [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+const {
+    GOOGLE_FONTS,
+} = SERVICE_STRING;
```
Comment (EngrEric) [inline components/Article/ArticlePage.js:18]: `const { GOOGLE_FONTS } = SERVICE_STRING;`

File: components/Article/ArticlePage.js [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+const {
+    ARTICLE_READ,
+    READ_MORE,
+} = HOME_STRINGS;
```
Comment (EngrEric) [inline components/Article/ArticlePage.js:23]: `const { ARTICLE_READ, READ_MORE } = HOME_STRINGS;`

File: components/Article/ArticlePage.js [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+                    <p className="article-read">
+                        {ARTICLE_READ}
+                        <a className="article-content-a" href="/articlePage">
+                            {READ_MORE}
+                                ...
+                        </a>
+                    </p>
+                </div>
+                <div className="article-content">
+                    <p className="article-content-header">What Should be in a First Aid kit?</p>
```
Comment (EngrEric) [inline components/Article/ArticlePage.js:59]: move all strings to constants and map over all these. always practice DRY (do not repeat yourself) when possible.

File: components/Service/ServicePage.js [hunk 11]
```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/jsx-indent-props */
+import React from 'react';
+import Head from 'next/head';
+import NavigationBar from '../Common/NavigationBar';
+import './service.scss';
+import { HOME_STRINGS } from '../constants';
+import { SERVICE_STRING } from './constants';
```
Comment (EngrEric) [inline components/Service/ServicePage.js]: merge these constants to same place

**Review by EngrEric — CHANGES_REQUESTED** [review]
clean up your code and also resolve the conflicts

File: components/Service/ServicePage.js (hunk 4 above)
Comment (Way2nnadi) [reply components/Service/ServicePage.js]: Do that now in this PR @Merdoth, practice writing cleaner up.

File: components/Article/ArticlePage.js (hunk 10 above)
Comment (Merdoth) [reply components/Article/ArticlePage.js:59]: Those are temporary headers that will be written

File: components/Service/ServicePage.js (hunk 11 above)
Comment (Merdoth) [reply components/Service/ServicePage.js]: Like you said I should practice DRY so I can't be repeating the same string that already exists when I can just import it from the initial place and if you pay enough attention to details you'll notice that those two constants are serving two very different purpose

**Pull request merged**